src/main/java/org/opensearch/knn/index/mapper/KNNVectorFieldMapperUtil.java

```diff
@@ -105,16 +105,17 @@ public static void validateVectorDataType(KNNMethodContext methodContext, Vector
         }
 
         if (VectorDataType.BYTE == vectorDataType) {
-            if (KNNEngine.LUCENE == methodContext.getKnnEngine()) {
+            if (KNNEngine.LUCENE == methodContext.getKnnEngine() || KNNEngine.FAISS == methodContext.getKnnEngine()) {
```

Comment:
better to put these engines in a list which says what engines support ByteVectors

Reply:
Ack